src/lib.rs

```diff
@@ -935,6 +936,24 @@ mod tests {
                 .unwrap();
             assert_eq!(context.stderr(), "+ echo 'foo bar'\n");
         }
+
+        #[test]
+        #[cfg(unix)]
+        fn arguments_with_invalid_utf8_will_be_logged_with_lossy_conversion() {
```

Comment:
I added this test, since now it's possible to create command arguments with invalid utf8.